app/main.py

```diff
@@ -1,8 +1,50 @@
+import decimal
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
```

Comment:
The line continuation character `\` is not needed here. You can simply place the `-> None:` on the same line as the method signature.